api/apps/api/test/scenario-input-files/scenario-cost-surface/scenario-cost-surface.e2e-spec.ts

```diff
@@ -1,19 +1,61 @@
-import { PromiseType } from 'utility-types';
-import { createWorld } from './world';
+import { FixtureType } from '@marxan/utils/tests/fixture-type';
+import { getFixtures } from './scenario-cost-surface.fixtures';
 
-let world: PromiseType<ReturnType<typeof createWorld>>;
+let fixtures: FixtureType<typeof getFixtures>;
 
 beforeAll(async () => {
-  world = await createWorld();
+  fixtures = await getFixtures();
+  await fixtures.GivenScenarioWasCreated();
+  await fixtures.GivenScenarioWithPuAndLocks();
 });
 
-describe(`When scenario has PUs with cost and lock status`, () => {
-  beforeAll(async () => {
-    await world.GivenScenarioWithPuAndLocks();
+describe(`As owner, when scenario has PUs with cost and lock status`, () => {
+  it(`returns relevant data for pu.dat`, async () => {
+    const result = await fixtures.WhenGettingMarxanDataAsOwner();
+    const [headers, ...costAndStatus] = result.split('\n');
+
+    expect(headers).toEqual('id\tcost\tstatus');
+    expect(costAndStatus).toMatchInlineSnapshot(`
```

Comment:
Wouldn't it be better to use .toEqual() ?
It seems clearer IMHO
```suggestion
    expect(costAndStatus).toEqual([
        "0	200	0",
        "1	400	2",
        "2	600	3",
        "3	800	0",
      ])
```

Reply:
To be honest, for most of those that use `MatchInLineSnapshot`, I just added variants without changing much the original tests done by Kamil. I guess this is a matter of preference, and yeah maybe it will be better if we use the same method for every test.

Reply:
I second @AaronPerezPerez recommendation.

Reply:
@rubvalave though I'm fine with either, I think the _matter of preference_ bit is mostly about workflow, not so much code style per se.

Snapshots (and the inline variant used throughout these tests) can be useful if wanting to reap some free labour from Jest, as they can be updated via `jest --updateSnapshot ----testNamePattern <pattern>` rather than having to manually edit the expected values for `toEqual()`/`toStrictEqual()` (the latter being what we likely want in most cases, tin-foil hat on).

I'd say that for small objects there is no practical difference. When checking asserting equality of complex objects such as a, say GeoJSON document, then snapshots can help substantively.